app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,28 @@
+module StripeWrapper
+  class Charge
+    attr_reader :error_message, :response
+
+    def initialize(options={})
+      @response = options[:response]
+      @error_message = options[:error_message]
+    end
+
+    def self.create(options={})
+      begin
+        response = Stripe::Charge.create(
+          amount: options[:amount],
+          currency: "usd",
+          source: options[:source],
+          description: options[:description]
```

Comment:
Sometimes you can see that the code is aligned like this:

```ruby
  amount:      options[:amount],
  currency:    "usd",
  source:      options[:source],
  description: options[:description]
```

Makes it slightly easier to read.